src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -526,7 +526,7 @@ namespace winrt::TerminalApp::implementation
         Controls::MenuFlyoutItem closeTabMenuItem;
         Controls::FontIcon closeSymbol;
         closeSymbol.FontFamily(Media::FontFamily{ L"Segoe MDL2 Assets" });
-        closeSymbol.Glyph(L"\xE8BB");
+        closeSymbol.Glyph(L"\xEDAE");

```

Comment:
[Segoe MDL2 icons](https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font#icon-list) mentions "EDAE ErrorBadge12". Using an error badge to represent closing a tab seems dubious to me, even if the glyph is currently similar.

Reply:
Then maybe I will try make the original icon smaller > <

Reply:
Either that or use “F13D StatusCircleErrorX”